### DISCUSSION TITLE
[Feature request]: get `cape-history` to work in vterm-mode

### DISCUSSION THREAD

**Discussion by prosoitos** [minad/cape#102] [Ideas]
I use `cape-history` in inferior programming modes that derive from comint (e.g. `inferior-python-mode`, `inferior-ess-r-mode`).

With Julia however, I use VTerm to run the Julia process and unfortunately, `cape-history` does not work in `vterm-mode`. Might this be something that could be added???

Thank you!!!!!

**Comment by minad** [comment]
Cape is mostly limited to builtin Emacs functionality, but if the addition is reasonably simple, why not? I don't use Vterm, so I am not sure what we would have to add.

**Reply by minad** [reply]
See also the related issue for Consult: https://github.com/minad/consult/issues/121
